packages/trpc/server/routers/viewer/eventTypes/getByViewer.handler.ts

```diff
@@ -245,11 +245,17 @@ export const getByViewerHandler = async ({ ctx, input }: GetByViewerOptions) =>
                 ? orgMembership
                 : membership.role,
             profile: {
-              image: getTeamAvatarUrl({
-                slug: team.slug,
-                requestedSlug: team.metadata?.requestedSlug ?? null,
-                organizationId: team.parentId,
-              }),
+              image: team.parentId
+                ? getOrgAvatarUrl({
+                    slug: team.parent?.slug || null,
+                    logoUrl: team.parent?.logoUrl,
+                    requestedSlug: team.slug,
+                  })
+                : getTeamAvatarUrl({
+                    slug: team.slug,
+                    requestedSlug: team.metadata?.requestedSlug ?? null,
+                    organizationId: team.parentId,
```

Comment:
Seems odd that we are in the else of the ternary that is getting a non-org URL but yet we are passing an organizationId. 

Is this still needed or can we refactor?

Reply:
Let me see what I can do

Reply:
We’re not necessarily in a non org situation - we’re just in a situation where a sub team has a org as a parent 🤔